src/AccordionItemHeading/AccordionItemHeading.tsx

```diff
@@ -12,6 +12,18 @@ type AccordionItemHeadingProps = React.HTMLAttributes<HTMLDivElement> & {
 
 type AccordionItemHeadingState = {};
 
+const getClosest = (el, selector) => {
+    return el && (el.matches(selector) ? el : getClosest(el.parentNode, selector));
+};
+
+const focusNextHeading = (evt, el) => {
+    if(el !== evt.target) {
```

Comment:
I'm not really sure that this conditional is necessary: there's no harming in calling `.focus` on the currently focussed element is there?

Reply:
Ah, was getting an error when there is only one single item in the accordion. The accordion had no first or last item and I was just putting a test in that reusable function since I was just repeating the same code in each case in the handlekeypress function. But as you pointed out markup not being strictly enforced and needs tweaking there, i'll test again